roles/pulp-web/tasks/main.yml

```diff
@@ -17,7 +17,6 @@
   k8s:
     state: "{{ deployment_state }}"
     definition: "{{ lookup('template', 'templates/' + item + '.deployment.yaml.j2') | from_yaml }}"
-    wait: true
```

Comment:
why remove this?

Reply:
Because now we are deploying pulp-web before pulp-api but the readiness check for pulp-web depends on being able to hit the status endpoint. The ‘wait: true’ causes processing to stop until pulp-web is “running”. We want it to proceed with the remainder of the deployments.

Reply:
thank you for the explanation!